tfx/components/util/tfxio_utils.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """TFXIO (standardized TFX inputs) related utilities."""
+from __future__ import annotations
```

Comment:
had to add this to allow for proto enums to be used as types in signatures. Not sure if this is something desirable from the tfx side or not @jiyongjung0 @1025KB

Reply:
I can go back to string in the `make_tfxio ` function signature instead of the proto enum type.